### DISCUSSION TITLE
Statistics

### DISCUSSION THREAD

**Discussion by rugric** [xcompact3d/Incompact3d#197] [Q&A] [answered]
Hello everyone. I wanted to know if the init_stat parameter that is inside the input.i3d can be different after restarting a simulation, or do I have to start the simulation from the beginning ( This because I realized that I entered a too small value, where the flow was still laminar). Thanks

**Comment by slaizet** [comment, marked as answer]
Hi,
I think you can change it (to a value higher than the current time step otherwise it will mess up the statistics)
Best,
Sylvain
